DigitalLearningSolutions.Web/ViewModels/TrackingSystem/CourseSetup/CourseDetails/EditCourseDetailsFormData.cs

```diff
@@ -0,0 +1,88 @@
+﻿namespace DigitalLearningSolutions.Web.ViewModels.TrackingSystem.CourseSetup.CourseDetails
+{
+    using System.ComponentModel.DataAnnotations;
+    using DigitalLearningSolutions.Web.Attributes;
+    using DigitalLearningSolutions.Web.Helpers;
+
+    public class EditCourseDetailsFormData
+    {
+        public EditCourseDetailsFormData() { }
+
+        public EditCourseDetailsFormData(

```

Comment:
This doesn't look like it is used anywhere outside the unit test helper method. If so I'd get rid of it and just use an initialiser with the empty contructor there.

Reply:
removed this and replaced its usage with an object initializer